cores/esp8266/Arduino.h

```diff
@@ -209,6 +209,7 @@ unsigned long millis(void);
 unsigned long micros(void);
 uint64_t micros64(void);
 void delay(unsigned long);
+void interruptable_delay(unsigned long ms);
```

Comment:
`interruptible_delay()` ?

Reply:
[Sorry, pushed changes with mistype before verify]
I've walk through delay() calls in esp8266/Arduino and found only two suspected delays both in mDNS related stuff. All other delays (in libraries code) defined by constants and not longer than 300mS (so they out of possible problems scope).